docusaurus/docs/cloud/advanced/email.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Email Provider
+displayed_sidebar: cloudSidebar
+description: Configure Strapi Cloud to use a third-party email provider.
+canonicalUrl: https://docs.strapi.io/cloud/advanced/email.html
+---
+
+# Email Provider
+
+Strapi Cloud comes with a basic email provider out of the box. However, you can also configure it to utilize a third-party email provider, if needed.
+
+:::prerequisites
+
+- A local Strapi project running on `v4.8.2+`.
+- Credentials for a third-party email provider (see [Strapi Market](https://market.strapi.io/)).
+
+:::
+
+## Configuration
+To configure a third-party email provider, in your Strapi project, create a `./config/env/production/plugins.js` or `./config/env/production/plugins.ts` file with the following content:
+
+```js
+module.exports = ({ env }) => ({
+  // ...
+  Required Email Provider Plugin Configuration
```

Comment:
I'd suggest we replace the whole block and its introduction sentence (lines 20-28) with the following:

> To configure a 3rd-party email provider in your Strapi project, create or edit the plugins configuration file for your production environment `./config/env/production/plugins.js|ts` by adding email configuration options as follows:

```js title="./config/env/production/plugins.js"
module.exports = ({ env }) => ({
// … some unrelated plugins configuration options
// highlight-start
email: {
   config: {
      // … provider-specific email configuration options go here
   }
// highlight-end
// … some other unrelated plugins configuration options
)}


FYI, the `highlight-start` and `highlight-end` comments won't show up on the website and are [one of the ways to highlight specific lines](https://docusaurus.io/docs/markdown-features/code-blocks#line-highlighting) in code examples with Docusaurus. 😊 

As mentioned by other reviewers, it'd actually be nice to provide both JavaScript and TypeScript examples, even if they only slightly differ (`export default ({ env }) => ({` in TS instead of module.exports).

When you'll create JS/TS tabs, please ensure that you name the `groupId` `js-ts` so these tabs sync with all other pages on the website. Here's an example (with nested tabs): https://github.com/strapi/documentation/edit/main/docusaurus/docs/dev-docs/configurations/server.md.